pkg/config/spec.go

```diff
@@ -47,6 +47,27 @@ const (
 	TiB
 )
 
+// TODO: Move it to the sdk
+func resolveTarget(fs v1.FS, target string) (string, error) {
```

Comment:
good here to test it directly on kairos via branches, if it works will move it into the sdk as its a very generic method that can be used everywhere, including immucore